blog/_posts/2023-01-24-scala-center-2023-roadmap.md

```diff
@@ -0,0 +1,247 @@
+Scala 2 and Scala 3 versions of code examples in the documentation
+([docs.scala-lang#2450](https://github.com/scala/docs.scala-lang/pull/2450)). We created an
+[issue](https://github.com/scala/docs.scala-lang/issues/2481) to coordinate the update of the whole documentation to
+always show both Scala 2 and Scala 3 code examples when possible.
+
+**We offered individualized support to our MOOCs’ learners.** We published the course “Effective Programming in Scala”
+to the [Extension School platform](https://www.extensionschool.ch/learn/effective-programming-in-scala). This
+platform allows the learners to have regular 1-on-1 meetings with our instructors along their learning journey. Learn
+more about the motivation in the previous [announcement]({% link blog/_posts/2022-06-08-learn-scala-at-epfl-extension-school.md %}).
+
+### Developer Experience
+
+**We improved the debugger for Scala 3.** The Metals’ debugger is now able to evaluate arbitrary Scala 3 expressions
+while the debugger is paused in the middle of a program. Step-by-step execution now automatically skips the methods
+generated by the compiler (e.g. mix-in forwarders, getters, setters, bridge methods, and synthetic methods of case
+classes). Learn more in [this video](https://www.youtube.com/watch?v=UePrOa_1Am8).
+
+**We fixed the apps installed by the [setup instructions](https://docs.scala-lang.org/getting-started).** The
+command-line apps `sbt` and `scala` now behave consistently regardless of the installation process (`cs install` vs
+manual download of the release artifacts).
+
+### Community and Contributor Experience
+
+**We co-organized the [ScalaCon](http://www.scalacon.org/) online conference.** We were in charge of the program and
+the program committee.
+
+**We published videos about Scala 3.** [Let’s Talk About Scala 3](https://www.youtube.com/playlist?list=PLTx-VKTe8yLxYQfX_eGHCxaTuWvvG28Ml)
+is a series of videos where we share interesting, useful, and cool things related to Scala 3. We published 4 new
+videos in 2022.
+
+**We improved the code editing experience in [Scastie](https://scastie.scala-lang.org).** Scastie is the online Scala
+code editor that is used every day by one thousand developers to share code snippets in the community. We have 
+implemented “IDE features” such as autocompletion and showing type information under the mouse pointer.
+
+**We mentored new contributors to the Scala 3 compiler.** Diving into the compiler is not an easy task. The Compiler
+Academy organizes online pair-programming events to mentor new contributors on the compiler codebase. You can learn
+more about the Compiler Academy in [this blog article]({% link blog/_posts/2022-11-02-compiler-academy.md %}).
+
+**We implemented the building blocks of a new tool to detect incompatibilities between programs.** Guaranteeing the
+absence of incompatibilities between library dependencies is the cornerstone of a peaceful Scala ecosystem. We have
```

Comment:
“peaceful” is a odd word here — I strongly associate that word with conflicts between humans, not incompatibilities between libraries. maybe simply omit that word, or reword the sentence.

Reply:
I replaced it with “seamless” in 6c333175, what do you think?

Reply:
that works